.github/workflows/update.yml

```diff
@@ -0,0 +1,14 @@
+name: "Update"
+on:
+  workflow_dispatch:
+  schedule: 
+    - cron: "21 4 * * 0"
```

Comment:
Any reason this doesn't run every day anymore?

Reply:
None in particular. Make it daily.